src/main/java/com/openmrs/migrator/core/utilities/KettleUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.openmrs.migrator.core.utilities;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KettleUtils {
+
+  private static Logger log = LoggerFactory.getLogger(KettleUtils.class);
+
+  private final String KETTLE_PROPERTIES = "kettle.properties";
+
+  private final String KETTLE_DIR = ".kettle";
+
+  public void loadProperties() {
+
+    try (InputStream input = getClass().getResourceAsStream("/structure/" + KETTLE_PROPERTIES)) {
+
+      Properties prop = new Properties();
+
+      log.info(
+          "Loading connection details from "
+              + KETTLE_PROPERTIES
+              + " file in the classpath to system properties");
+
+      prop.load(input);
+
+      log.info(
+          "Writing properties to <user home directory>/"
+              + KETTLE_DIR
+              + "/"
+              + KETTLE_PROPERTIES
+              + " file");
+      writePropertiesToKettlePropertiesOutter(this.getKettlePropertiesLocation(), prop);
+
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  private void writePropertiesToKettlePropertiesOutter(File file, Properties properties)
```

Comment:
This should get handled as part of ticket 85 unless you need it right now?

I'd rather this be moved to the setup or bootstrap step - but if needed for the PDI service to run then lets include there. It will not be used by anything else

Reply:
Well I  left it here  to enable the user to  run the app  without  typing  the  ETL_* options

Reply:
ok, I'm not sure I understand why this means we need a utils class for this - can you maybe elaborate further?

It seems to me that these are kettle properties that PDI requires to be able to run, which we are going to control from a settings file. We can then have the settingsService update the kettle.properties file where needed (seems to me this falls under ticket 100 - sorry I previously said 85, but I was referring to ticket 100).

If we need something now for PDI to run, these can be controlled directly from the kettle.properties file before we compile the app. We just edit it for our local setup.

Reply:
Now I think  I did  wrong implementation. Because KettleUtils.java passes  the configuration  from  src/main/resources/.../kettle.properties  to  ~//kettle/kettle.properties when  the  app runs as the settingsin the kettle file are required.

Reply:
Seeing this it really looks like  this is not needed at moment